core/trino-main/src/main/java/io/trino/execution/scheduler/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -1191,35 +1191,39 @@ private StageId getStageId(PlanFragmentId fragmentId)
 
         private void scheduleTasks()
         {
-            long speculativeTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
+            long standardTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
                     .filter(context -> !context.getNodeLease().getNode().isDone())
-                    .filter(context -> context.getExecutionClass() == SPECULATIVE)
+                    .filter(context -> context.getExecutionClass() == STANDARD)
                     .count();
 
-            long nonSpeculativeTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
+            long speculativeTasksWaitingForNode = preSchedulingTaskContexts.values().stream()
                     .filter(context -> !context.getNodeLease().getNode().isDone())
-                    .filter(context -> context.getExecutionClass() == STANDARD)
+                    .filter(context -> context.getExecutionClass() == SPECULATIVE)
                     .count();
 
             while (!schedulingQueue.isEmpty()) {
-                if (nonSpeculativeTasksWaitingForNode >= maxTasksWaitingForNode) {
-                    break;
```

Comment:
in `Keep executionClass explicit in SchedulingQueue and PrioritizedSchedu…` where did it go to?

not sure if this commit is supposed to be a pure syntatical refactor or something more?

Reply:
It changes internal model of ScheulingQueue a bit. Not behavioral changes. Added some comment in commit message.